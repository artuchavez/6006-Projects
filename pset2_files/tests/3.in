max-heap
Abby,Betty,Cathy,Demi,Elly
5,3,4,2.3,1.7
Betty,Abby,Elly
1.6,3.6,5
