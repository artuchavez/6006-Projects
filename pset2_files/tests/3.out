Abby,Betty,Cathy,Demi,Elly
3.6,1.6,4,2.3,5